references/cli/adopt.go

```diff
@@ -440,6 +441,19 @@ func NewAdoptCommand(f velacmd.Factory, streams util.IOStreams) *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(o.Complete(f, cmd, args))
 			cmdutil.CheckErr(o.Validate())
+			if o.AppName != "" {
+				app, err := f.AppGetter().Get(o.AppName)
```

Comment:
Does `namespace` parameter should be specified ,when get Application?
Applications in different NS may have the same name.

Reply:
Yes, the namespace should be specified when getting an application because applications with the same name in different namespaces are considered separate entities.